azure-pipelines.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       Release:
         BuildConfiguration: Release
         _debugArg: ''
+  timeoutInMinutes: 90
```

Comment:
The timeout problem has been solved in another way and it is on master now, so I would revert this and rebase instead. And maybe squash as well so we don't get 20 new commits for this change.